### DISCUSSION TITLE
Edit a product in RelationManager, dividing price by 100

### DISCUSSION THREAD

**Discussion by terjelundemotangen** [filamentphp/filament#9826] [Help] [answered]
An Event got many products.
In the edit page for Event I got a ProductsRelationManager. From this I'm trying to edit one product.
Edited data is stored correctly. "price" is loading correct data (in cents).
My problem is loading the "price" divided by 100 into text input "price_to_save".

```
Hidden::make('price')
  ->required(),
TextInput::make('price_to_save')
  ->label(__('Price to save'))
  ->numeric()
  ->live(onBlur: true)
  ->dehydrated(false)
   ->afterStateUpdated(function (Set $set, $state) {
        $set('price', $state * 100);
     }),
```

In App\Models\Product:

```
protected function priceToSave(): Attribute
  {
    return Attribute::make(
      get: fn () => $this->price / 100,
    );
  }
```

```
  protected $casts = [
    'price' => 'integer',
  ];
```
Hope someone has a solution to this

**Comment by leandrocfe** [comment, marked as answer]
[afterStateHydrated](https://filamentphp.com/docs/3.x/forms/advanced#field-hydration) ?

**Reply by terjelundemotangen** [reply]
That easy 🫣. Thank you @leandrocfe
